helix-term/src/keymap.rs

```diff
@@ -593,6 +593,9 @@ impl Default for Keymaps {
             // paste_all
             "P" => paste_before,
 
+            "q" => record_macro,
+            "Q" => play_macro,
```

Comment:
I think this should be the reverse, `Q` should record macro and `q` should play macro like in vim.

Since it's a macro, it's expected to play more than record. And since play is expected to be used more frequently, it would be best use without extra shift.

Reply:
Good point, kakoune has this as well. I didn't realize it during review

Reply:
In Vim, record is `q` and play is `@`; I changed it here because I figured it's more consistent. But yeah, swapping `q` and `Q` here does make sense.

Reply:
(it's also "replay" instead of "play" but I didn't have strong opinions there)

Reply:
Oh, yeah sorry if I get stuff inconsistent with Kakoune, I've never used it and am not very familiar with it. Though in this situation, I do believe "play" makes very slightly more sense than "replay", since it is possible to play a macro without actually recording it.